scss/_variables.scss

```diff
@@ -755,6 +755,7 @@ $navbar-dark-brand-hover-color:           $navbar-dark-active-color !default;
 $dropdown-min-width:                10rem !default;
 $dropdown-padding-y:                .5rem !default;
 $dropdown-spacer:                   .125rem !default;
+$dropdown-color:                    $body-color !default;
```

Comment:
`$body-color` was already set on the dropdown component, so I moved to the config.